src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java

```diff
@@ -48,19 +48,24 @@ public class GuiManager implements Manager, GameScreenAPI {
     public final Gui quests;
     public final Gui minimap;
     public final Gui targetedOffers;
+    public final Gui monthlyDeluxe;
     public final LogoutGui logout;
     public final Gui eventProgress;
     public final Gui eternalGate;
     public final Gui blacklightGate;
+    public final Gui astralGate;
+    public final Gui astralSelection;
     public final RefinementGui refinement;
     public final PetManager pet;
     public final OreTradeGui oreTrade;
     public final GroupManager group;
 
     private LoadStatus checks = LoadStatus.WAITING;
     private enum LoadStatus {
-        WAITING(gm -> gm.quests.lastUpdatedOver(5000) && gm.quests.visible),
-        MISSION_CLOSING(gm -> gm.quests.show(false)),
+        WAITING(gm -> (gm.quests.lastUpdatedOver(5000) && gm.quests.visible)
+                || (gm.monthlyDeluxe.lastUpdatedOver(5000) && gm.monthlyDeluxe.visible)),
+        MISSION_CLOSING(gm -> gm.quests.visible && gm.quests.show(false)),
+        MONTHLY_CLOSING(gm -> gm.monthlyDeluxe.visible && gm.monthlyDeluxe.show(false)),
```

Comment:
Quests may not be visible at all (if you do not have an active running quest, it won't open).
You should not be waiting until they're visible to pass thru the state and allow it to move forward to click ammo once.

Reply:
ahhh I didn't know it gets stuck 😓